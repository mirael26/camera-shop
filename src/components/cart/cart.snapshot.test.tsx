import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import { productsInCartMock } from '../../test/mocks';
import Cart from './cart';

jest.mock('./cart-product-card/cart-product-card', () => 'CartProductCard');
jest.mock('./cart-summary/cart-summary', () => 'CartSummary');

test('Cart component renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<Cart/>, { initialState: { cart: { productsInCart: productsInCartMock}}});
  expect(asFragment()).toMatchSnapshot();
});
