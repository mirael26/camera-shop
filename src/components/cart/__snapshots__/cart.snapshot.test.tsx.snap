// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart component renders correctly 1`] = `
<DocumentFragment>
  <section
    class="basket"
  >
    <div
      class="container"
    >
      <h1
        class="title title--h2"
      >
        Корзина
      </h1>
      <ul
        class="basket__list"
      >
        <cartproductcard
          product="[object Object]"
        />
        <cartproductcard
          product="[object Object]"
        />
      </ul>
      <cartsummary />
    </div>
  </section>
</DocumentFragment>
`;
